Dockerfile

```diff
@@ -58,7 +58,7 @@ ENV APP_NAME=${APP_NAME} \
 # Update kernel and install runtime dependencies
 RUN apk --no-cache update && \
     apk --no-cache upgrade && \
-    apk --no-cache add ncurses-libs openssl bash erlang-crypto libssl1.1 yaml-dev nodejs
+    apk --no-cache add bash openssl erlang-crypto nodejs yaml-dev
```

Comment:
I wonder what is the change that doesn't require `libssl1.1` anymore? Is there a difference in execution from the docker-compose vs the way it is executed on by AWS or Heroku?

Reply:
Since both are running containers, there should not be any differences.

However, I also changed the dependencies on the _build_ step so SSL libraries might be compiled differently when the image is built… I’m going to setup a deployment on Heroku to give it a try!